ibm_catalog.json

```diff
@@ -0,0 +1,216 @@
+          },
+          {
+            "title": "Supports application build",
+            "description": "Supports application build and configuration using dedicated repositories"
+          },
+          {
+            "title": "Supports Service To Service authorisation policy creation to a MQ instance",
+            "description": "Supports Service To Service authorisation policy creation to a MQ instance to allow the Enterprise Application Service for Java to connect to this"
+          }
+        ],
+        "flavors": [
+          {
+            "label": "Standard",
+            "name": "standard",
+            "install_type": "fullstack",
+            "working_directory": "solutions/standard",
+            "compliance": {
+              "authority": "scc-v3",
+              "profiles": []
+            },
+            "iam_permissions": [
+              {
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::role:Editor"
+                ],
+                "service_name": "enterprise-application-service"
+              }
+            ],
+            "architecture": {
+              "descriptions": "This architecture creates and configures an instance of Enterprise Application Service for Java",
+              "features": [
+                {
+                  "title": "Creates an instance of Enterprise Application Service for Java",
+                  "description": "Creates and configures an instance of Enterprise Application Service for Java"
+                },
+                {
+                  "title": "Supports application build",
+                  "description": "Supports application build and configuration using dedicated repositories"
+                },
+                {
+                  "title": "Supports Service To Service authorisation policy creation to a MQ instance",
+                  "description": "Supports Service To Service authorisation policy creation to a MQ instance to allow the Enterprise Application Service for Java to connect to this"
+                }
+              ],
+              "diagrams": [
+                {
+                  "diagram": {
+                    "caption": "Creates and configures an instance of Enterprise Application Service for Java",
+                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-ease/draft_da/reference-architecture/deployable-architecture-ease.svg",
+                    "type": "image/svg+xml"
+                  },
+                  "description": "This architecture creates and configures an instance of Enterprise Application Service for Java"
+                }
+              ]
+            },
+            "configuration": [
+                {
+                  "key": "ibmcloud_api_key",
+                  "required": true
+                },
+                {
+                  "key": "resource_group"
+                },
+                {
+                  "key": "region",
```

Comment:
how about using the general IBM Cloud region picker so we don't have to main this? For syntax, see https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/blob/2218c2422de36b1dfa771082ff06de1d2c3aec68/ibm_catalog.json#L119-L130

Reply:
keeping single region while waiting for availability to be extended